src/components/map/OlMap.vue

```diff
@@ -299,11 +317,14 @@
                     new ol.control.Attribution()
                 ],
 
-                layers: this.mapLayers.concat(this.dataLayers).concat([
-                    this.biodivLayer,
+                layers: [
```

Comment:
Good syntax, thank you. I'm definitlty a rookie :)

Reply:
Nah, it's only some syntaxic sugar allowed with es6. But I like it because it is more consise.